src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -25,6 +25,7 @@
     "require-dev": {
         "symfony/asset": "~3.4|~4.0",
         "symfony/browser-kit": "~3.4|~4.0",
+        "symfony/config": "~4.2",
```

Comment:
I think this can be a normal requirement instead. I don't think it is currently possible to avoid it anyway.